doto-backend/index.js

```diff
@@ -4,12 +4,30 @@ const cors = require('cors')
 const app = express()
 const apiPort = 3000
 
+// Mongoose connection
+const mongoose = require('mongoose');
+const connectionString = "mongodb+srv://701Group2:SoftwareEngineering701Group2@cluster0-ahuai.mongodb.net";
```

Comment:
Can we use an env for this? So that we can mask the username and password.

Reply:
yeah, done :)